docs/src/main/sphinx/connector/hive-security.rst

```diff
@@ -218,8 +201,20 @@ The default authentication type for the Hive metastore is ``NONE``. When the
 authentication type is ``NONE``, Trino connects to an unsecured Hive
 metastore. Kerberos is not used.
 
-Example configuration with ``KERBEROS`` authentication
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+``NONE`` authentication with impersonation MISSING
+""""""""""""""""""""""""""""""""""""""""""""""""""
+
+>>>MISSING<<<
```

Comment:
What is this?

Reply:
Missed in review! It was my intent to ask during reviews if these two options, which exist for kerberos, do not exist for NONE, and create a task for myself to come back to it in a later PR if they do to complete documenting these options, as this PR is not intended to introduce any new technical content. (And of course, clean up, which I will do post haste and squash and merge.)

Reply:
Removed and made note of possibly missing configurations in my hive docs reorganization notes.